till_korten/devbio-napari_cluster/readme.md

```diff
@@ -0,0 +1,241 @@
+
+<img src="images/1_login.png" width="500" />
+
+
+Afterwards, you should see a single button `Start My Server`. Click on it:
+
+<img src="images/2_start_server.png" width="500" />
+
+Now you get to configure the computing node you want your session to run on. Switch to the advanced configuration by clicking the button `Advanced`. Then you should see something like the image below. 
+
+1. Start by choosing a preset (click on 1). 
+2. You should choose a GPU node preset (2). You can choose between
+   * Ampere A100 -> more powerful than any gaming GPU, 40GB graphics memory. However these machines are often more used and it may take longer to get a node on the cluster
+   * Tesla K80 -> half as as powerful as a GTX 1080, but 12GB graphics memory. This is useful for testing whether your workflows work on a GPU on the cluster and for tasks that are not time critical.
+
+Choose according to the your needs and the current utilization (the bars above the preset chooser) (the A100 are on the bar labeled `alpha`, the K80 are on the bar labeled `gpu2`) If the partition is very full, you may have to wait a long time or not get a session at all. Choose the number of CPUs and memory per cpu as needed. Note that the memory is per CPU, so if you choose more CPUs, you get more memory. It is recommended to increase the number of CPUs rather than increasing the default value for memory per CPU because the default memory per CPU evenly divides the available memory to all CPUs (meaning that increasing mem-per-CPU effectively blocks more CPUs anyways).
+
+Once you are happy with your configuration, click the orange button `Spawn` at the very bottom.
+
+<img src="images/3_configure_node.png" width="500" />
+
+You will now see a wait bar. Do not worry if it does not move, this bar is always at 50%. It usually takes 2-5 min to get a node.
+
+<img src="images/4_wait.png" width="500" />
+
+## Step 3: Open a terminal
+
+Open a terminal by clicking on file (1 in the image below) -> new (2) -> Terminal (3)
+
+<img src="images/5_open_terminal.png" width="500" />
+
+## Step 4: Install a custom jupyter kernel for your user
+
+In the terminal, execute the following code:
+
+```bash
+git clone https://gitlab.mn.tu-dresden.de/bia-pol/singularity-devbio-napari.git
+```
+You will be asked for a user name and password. Enter the following:
+
+User name: `git_pull`
+
+Password: `Bc6Yg8k5MyZzURNuWChR`
+
+Once the download has finished, enter the following:
+
+```bash
+cd singularity-devbio-napari
+./install.sh <version>
+```
+
+replace `<version>` with the version of our devbio-napari environment you want to use. At the time of writing, the latest version was `v0.1.5`. Always specify a version (or a tag name you got from us in case we created a custom environment for you). This will ensure that your code is always executed in the same environment and your data analysis is reproducible. Note down the version you use in your lab book!
```

Comment:
```suggestion
replace `<version>` with the version of our devbio-napari environment you want to use. At the time of writing, the latest version was `v0.1.5`. Always specify a version (or a tag name you got from us in case we created a custom environment for you). This will ensure that your code is always executed in the same environment and your data analysis is reproducible. 

*Note*: In order to ensure [repeatability ](https://en.wikipedia.org/wiki/Repeatability) and [reproducibility](https://en.wikipedia.org/wiki/Reproducibility) of the results ou obtained from using a singularity container, we strongly recommend and encourage you to keep track of the used versions. This way, you can ensure the integrity of your analysis workflow further down the line.
```